Tools/ecl_ekf/process_logdata_ekf.py

```diff
@@ -99,3 +95,12 @@ def is_valid_directory(parser, arg):
 
 if not args.no_plots:
     print('Plots saved to {:s}.pdf'.format(args.filename))
+
+# print master test status to console
+if (test_results['master_status'][0] == 'Pass'):
+    print('No anomalies detected')
+elif (test_results['master_status'][0] == 'Warning'):
+    print('Minor anomalies detected')
+elif (test_results['master_status'][0] == 'Fail'):
```

Comment:
What do you think about also printing the relevant level failures as we go? Warnings in yellow, failure in red.

Reply:
ok, I'm added a commit that does that now. It prints failures for the checks that can fail and warnings for the imu checks (can't fail). Atm it doesn't print warnings for the checks that can have both, but from my experience this doesn't occur anyways with the current threshold levels.